backend/packageLibrary.go

```diff
@@ -88,6 +88,9 @@ func UninstallApp(pkgId string) error {
 	task := installer.CreateUninstallTask(pkgId)
 	return task.Uninstall()
 }
+func Cancel(pkgId string) {
```

Comment:
make function name descriptive. instead of cancel make this cancelInstall. names can have many meetings make it descriptive as possible